### PR TITLE
Setup Contributing guide

### PR DESCRIPTION
**Pull request by tarang9211**
@Monte9 @binoy14 @bozzmob PR to set up contributing guide.

### CODE REVIEW

**Comment by Monte9** [comment]
@tarang9211 Thanks on the PR! 👍 

Let's start by changing the base to `v0.11.0` and fix any merge conflicts that come up.

**Comment by tarang9211** [comment]
> Let's start by changing the base to v0.11.0 and fix any merge conflicts that come up.

what do you mean? I pulled the latest from `v0.11.0` branch and created a branch off of that.

**Comment by Monte9** [comment]
Is this still WIP? Or is it ready to be reviewed?

**Comment by tarang9211** [comment]
WIP @Monte9 Let's add a WIP label.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### Solution
+
+What needs to be done to address this issue? Ideally, provide a pull request with a fix.
+
+### Additional Information
+
+* React Native version: [FILL THIS OUT: Be specific, filling out "latest" here is not enough.]
+* Platform: [FILL THIS OUT: iOS, Android, or both?]
+* Development Operating System: [FILL THIS OUT: Are you developing on MacOS, Linux, or Windows?]
```
Comment (binoy14) [inline .github/ISSUE_TEMPLATE.md]: `Development Operating System` and `Dev Tools` are not really useful for us to debug. Thoughts @Monte9

File: .github/CONTRIBUTING.MD [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+If you would like to submit a feature request or bug report, we encourage you to first look through the [issues](https://github.com/react-native-training/react-native-elements/issues) and [pull requests](https://github.com/react-native-training/react-native-elements/pulls) before filing a new issue.
+
+# Setup
+
+*  Install [Node.js](https://nodejs.org/) if you have not already.
+  *We advice you to use node 6.x.x*
+*  Fork the **react-native-elements** repo at (https://github.com/react-native-training/react-native-elements)[https://github.com/react-native-training/react-native-elements]
+* `git clone <your-forked-repo-url> && cd react-native-elements`
+* `yarn install` OR `npm install`
+* `npm test`
```
Comment (binoy14) [inline .github/CONTRIBUTING.MD]: Providing a link to https://github.com/react-native-training/react-native-hackathon-starter as a starter template would be useful

File: .github/CONTRIBUTING.MD [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+Thank for taking the time to checkout the repo and willing to contribute!
```
Comment (Monte9) [inline .github/CONTRIBUTING.MD]: `Thank for taking the time to check out the repo and be willing to contribute!`

File: .github/CONTRIBUTING.MD [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Feature requests, bugs, and code improvements are welcome as contributions!
+
+If this is your first open source contribution, please have a look at [this](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) guide.
```
Comment (Monte9) [inline .github/CONTRIBUTING.MD]: `please take a look at`

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Monte9) [inline .github/ISSUE_TEMPLATE.md]: > I think the rest is watever. And the last part is good but for our purposes it doesnt seem too necessary.

I would argue we should can remove `Additional Information` section

Comment (binoy14) [inline .github/ISSUE_TEMPLATE.md]: True, not really needed. Maybe just add react-native-elements version?

**Comment by tarang9211** [comment]
Sweet, I'll take a look at these changes and have them ready 👍

**Pull request merged**